### PR TITLE
[zh] Update container-lifecycle-hooks.md

### PR DESCRIPTION
**Pull request by gaogao101**
Signed-off-by: Gao Qian <gaoq.jy@fujitsu.com>

### CODE REVIEW

File: content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md [hunk 1]
```diff
@@ -197,15 +197,15 @@ The logs for a Hook handler are not exposed in Pod events.
...
+To generate a failed `FailedPostStartHook` event yourself, modify the [lifecycle-events.yaml](https://raw.githubusercontent.com/kubernetes/website/main/content/en/examples/pods/lifecycle-events.yaml) file to change the postStart command to "badcommand" and apply it.
 Here is some example output of the resulting events you see from running `kubectl describe pod lifecycle-demo`:
 -->
 ### 调试回调处理程序
 
 回调处理程序的日志不会在 Pod 事件中公开。
 如果处理程序由于某种原因失败，它将播放一个事件。
 对于 `PostStart`，这是 `FailedPostStartHook` 事件，对于 `PreStop`，这是 `FailedPreStopHook` 事件。
-要自己生成失败的 `FailedPreStopHook` 事件，请修改
+要自己生成失败的 `FailedPreStartHook` 事件，请修改
```
Comment (windsonsea) [inline content/zh-cn/docs/concepts/containers/container-lifecycle-hooks.md:208]:
```suggestion
要自己生成失败的 `FailedPostStartHook` 事件，请修改
```
这都能贴错吗？

**Review by windsonsea — CHANGES_REQUESTED** [review]

**Comment by gaogao101** [comment]
@windsonsea I'm really sorry. I won't make such a simple mistake again next time. Thank you for your correction.

**Pull request merged**